Subject: Small mistake in documentation
From: Bart Martens <bartm@knars.be>
Date: Sun, 11 Jun 2006 19:11:18 +0200
X-Forwarded: http://rt.cpan.org/Ticket/Display.html?id=48007

Minor typo in manpage, reported by Gabor Kiss <kissg@cdata.hu>, see #327783.

--- a/lib/Text/Glob.pm
+++ b/lib/Text/Glob.pm
@@ -143,7 +143,7 @@
 
 =item C<?> - match exactly one character
 
-C<a?> matches C<aa>, but not C<a>, or C<aa>
+C<a?> matches C<aa>, but not C<a>
 
 =item Character sets/ranges
 
